src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -27,15 +27,19 @@ export default function * loadCityContent (database: MemoryDatabase, newCity: st
   database.categoriesMap = categoriesMap
   database.languages = languages
 
-  const resourceCache: ResourceCacheType = yield call(fetchResourceCache, newCity, newLanguage, {
+  const resourceCache: ResourceCacheStateType = yield call(fetchResourceCache, newCity, newLanguage, {
     ...categoryUrls,
     ...eventUrls
   })
 
   database.addCacheEntries(resourceCache)
 
-  yield call(() => database.writeCategories())
-  yield call(() => database.writeResourceCache())
-  yield call(() => database.readCategories())
-  yield call(() => database.readResourceCache())
+  console.dir(database.categoriesMap)
+
+  // yield call(() => database.writeCategories())
```

Comment:
Why did you comment this out?

Reply:
This is just a test usage and probably should happen in a different way in the future.

Reply:
Add a comment for that please

Reply:
done in NATIVE-140